source/javascripts/controllers/_StackController.js.erb

```diff
@@ -36,11 +36,18 @@
 					}
 					return machineType.stackType === stack.type;
 				});
-				result[stack.id] = _.groupBy(stackMachineTypes, 'tag');
```

Comment:
Oh, I didn't realize we were already grouping! I checked, and currently the only values for `tag` are `gen1` and `gen2`, so I'm confident there's no downside here.